app/_hub/appsentinels/appsentinels/schemas/_index.json

```diff
@@ -0,0 +1,270 @@
+{
+  "fields" : [
+      {
+         "protocols": {
+         "elements": {
+            "one_of": [ 
+               "http",
+               "https"
+            ],
+            "type": "string"
+         },
+         "required": true,
+         "type": "set",
+         "default": [
+            "http",
+            "https"
+         ]
+         }
+      },
+      {
+        "config" : {
+           "type" :"record",
+           "fields" : [
+              {
+                 "http_endpoint" : {
+                    "type" : "string",
+                    "required": true
+                 }
```

Comment:
Can you please add descriptions for each field under `config`? Currently, the generated config page looks like this: https://deploy-preview-6778--kongdocs.netlify.app/hub/appsentinels/appsentinels/configuration/

When you add descriptions in this file, it'll look something like this: https://deploy-preview-6778--kongdocs.netlify.app/hub/kong-inc/ldap-auth/configuration/

For example, for `http_endpoint` it would look something like this:
```json
"http_endpoint" : {
     "type" : "string",
      "required": true,
      "description": "A string representing a host name, such as example.com."
}
```

Reply:
Thanks! Have updated the schema with descriptions.